srcpkgs/Bear/template

```diff
@@ -12,7 +12,7 @@ license="GPL-3.0-or-later"
 homepage="https://github.com/rizsotto/Bear"
 changelog="https://raw.githubusercontent.com/rizsotto/Bear/master/ChangeLog.md"
```

Comment:
looks like the changelog was removed sometime between 2.4.4 and 3.0 :/

Reply:
You are right, sorry didn't check the links. 

Should I simply remove it ? Didn't find an alternative in the Bear git page

Reply:
yeah, doesn't seem like they're doing a changelog anymore